File: server/Models/Ingredient.cs
using System.ComponentModel.DataAnnotations;

namespace all_spice_dotnet.Models;

public class Ingredient
{
  public int Id { get; set; }
  public DateTime CreatedAt { get; set; }
  public DateTime UpdatedAt { get; set; }
  public int RecipeId { get; set; }
  [MinLength(0), MaxLength(255)] public string Name { get; set; }
  [MinLength(0), MaxLength(255)] public string Quantity { get; set; }
  public Profile Creator { get; set; }
}

File: client/src/models/Ingredient.js


export class Ingredient{
  constructor(data){
this.id = data.id
this.created_at = new Date(data.created_at)
this.updated_at = new Date(data.updated_at)
this.name = data.name
this.quantity = data.quantity
this.recipe_id = data.recipe_id
  }
}